tests/suites/test_suite_nist_kw.function

```diff
@@ -14,6 +14,107 @@ void mbedtls_nist_kw_self_test( )
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_AES_C */
+void mbedtls_nist_kw_basic_tests( )
```

Comment:
What's the point of these basic tests? How do they differ from the self-tests in `nist_kw.c`?

Reply:
I agree, but I couldn't think of a proper name.
What do you suggest? Basic functionality test? multi context multi mode test?

Reply:
How about `mbedtls_nist_kw_mix_contexts`?

Reply:
OK